modules/nw-networkpolicy-about.adoc

```diff
@@ -120,6 +120,31 @@ spec:
       port: 443
 ----
 
+* Accept connections by using both namespace and Pod selectors:
++
+To match network traffic by combining namespace and Pod selectors, you can use a NetworkPolicy object similar to the following:
++
+[source,yaml]
+----
+kind: NetworkPolicy
+apiVersion: extensions/v1beta1
+metadata:
+  name: allow-pod-and-namespace-both
```

Comment:
snake style would be preferred here ```allow_pod_and_namespace_both```

Reply:
@anuragthehatter, is there an OpenShift convention for this? The dashes originated with the OCP 3.x docs that I ported for 4.1.

Reply:
@jboxman Thats fine. I thought you referred from the bug description where i pasted this object. I am not sure of any convention though.

Reply:
@anuragthehatter, we've been using `-` somewhat consistently throughout the docs in YAML. I found 572 instances of a `-` in a `name` key across 283 files in `master`. Ideally, I'd like to keep using `-` in the examples for `NetworkPolicy`.